src/main/resources/xlr/CreateAndStartSubRelease.py

```diff
@@ -9,6 +9,11 @@
 
 from xlr.XLReleaseClientUtil import XLReleaseClientUtil
 
+def find_planned_gate_task(tasks):
```

Comment:
Actually.. You hit the nail on the head :-)

I tried checking for a gate task but couldn't find out how. Do you know how?

The 'not found' case is handled in the main flow, throwing an error

Regards,
Vincent
(929) 240 7243<tel:(929)%20240%207243>

On Feb 19, 2016, at 12:45 AM, Andrew Phillips <notifications@github.com<mailto:notifications@github.com>> wrote:


In src/main/resources/xlr/CreateAndStartSubRelease.py<https://github.com/xebialabs-community/xlr-xlrelease-plugin/pull/13#discussion_r53402115>:

> @@ -9,6 +9,11 @@
>
>  from xlr.XLReleaseClientUtil import XLReleaseClientUtil
>
> +def find_planned_gate_task(tasks):


[minor] Bit of a confusing name? The method actually just finds planned tasks, it doesn't check that the task is a gate. Also, what if there is no planned gate task?

-
Reply to this email directly or view it on GitHub<https://github.com/xebialabs-community/xlr-xlrelease-plugin/pull/13/files#r53402115>.